src/Maui/Prism.Maui/Navigation/Builder/INavigationBuilder.cs

```diff
@@ -12,7 +12,9 @@ public interface INavigationBuilder
     INavigationBuilder UseAbsoluteNavigation(bool absolute);
     INavigationBuilder UseRelativeNavigation();
 
+    [Obsolete($"Use {nameof(GoBackAsync)} instead.")]
     Task<INavigationResult> GoBackTo<TViewModel>(INavigationParameters parameters);
```

Comment:
Since this is not part of an officially released API and only in a preview releases we can safely just remove the Obsolete API

Reply:
ok, I have removed all obsolete methods